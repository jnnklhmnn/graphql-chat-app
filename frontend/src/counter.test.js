import React from 'react';
import Counter from './Counter.js';
import renderer from 'react-test-renderer';

test('Counter should render without crashing', () => {
  const component = renderer.create(
    <Counter numberOfMessages={3}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});