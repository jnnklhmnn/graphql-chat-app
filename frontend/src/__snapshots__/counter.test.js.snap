// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should render without crashing 1`] = `
<div
  className="counter"
>
  <h2>
    3 items
  </h2>
</div>
`;
